src/test/java/no/uib/inf112/desktop/TestGraphics.java

```diff
@@ -21,6 +21,7 @@
     @BeforeClass
     public static void init() {
         // Note that we don't need to implement any of the listener's methods
+        Main.HEADLESS = true;
         application = new HeadlessApplication(new ApplicationListener() {
```

Comment:
Maybe refactor to use Mockito for Application and GL while we're at it?

Reply:
I don't initially understand what you mean, could you describe it more? maybe on slack?

Reply:
Do we need mockito isn't uglier than what we have now with no other benefit?

Reply:
You call `Mockito.mock(ApplicationListener.class)` (removing 25-ish lines) and set `Gdx.gl20` to `Mockito.mock(GL20.class)`  (removing 800-ish lines). In the end making the class way more readable and way way shorter (from 875 lines to 38 lines)